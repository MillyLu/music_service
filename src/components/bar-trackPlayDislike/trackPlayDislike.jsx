import { useContext} from 'react';
import { ThemeContext } from '../../ThemeProvider';
import * as Styled from './styles';

export function TrackPlayDislike({handleDeleteFavorite}){
    const {theme} = useContext(ThemeContext);
    return(
        <Styled.TrackPlayDislike>
            <Styled.TrackPlayDislikeSvg alt='dislike' theme={theme} onClick={handleDeleteFavorite}>

            </Styled.TrackPlayDislikeSvg>
        </Styled.TrackPlayDislike>
    )
}
