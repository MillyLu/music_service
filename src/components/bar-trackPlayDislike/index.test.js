/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
// import { ShallowRenderer, createRenderer } from 'react-shallow-renderer';
import { TrackPlayDislike } from './trackPlayDislike';



describe('<TrackPlayDislike/>', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<TrackPlayDislike />);
            const result = renderer.getRenderOutput();
            expect(result).toMatchSnapshot();
          })
      });
