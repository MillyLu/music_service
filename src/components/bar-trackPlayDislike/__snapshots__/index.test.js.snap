// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackPlayDislike/> should render successfully and match snapshot 1`] = `
<Styled(styled.div)>
  <Styled(Component)
    alt="dislike"
    theme={0}
  />
</Styled(styled.div)>
`;
